src/tailwind.preset.js

```diff
@@ -3,6 +3,7 @@ import plugin from 'tailwindcss/plugin.js';
 import colors from 'tailwindcss/colors.js';
 
 import focusVisibleRing from './tailwind.focus-visible-ring.js';
+import scrollShadows from './tailwind.scroll-shadows.js';
```

Comment:
Are we required to make these default exports here? If we could follow our usual convention of avoiding default exports except for Preact component modules with a single export, I think that would be preferable.